### PR TITLE
Add payment motion support

### PR DESCRIPTION
**Pull request by willm30**
This pr adds support for the payment motion. Test with: https://github.com/JoinColony/colonyCDapp/pull/430/

### CODE REVIEW

File: src/handlers/motions/motionCreated/helpers.ts [hunk 1]
```diff
@@ -14,22 +13,25 @@ import {
...
   const motion = await votingClient.getMotion(motionId);
 
-  try {
-    return colonyClient.interface.parseTransaction({
-      data: motion.action,
-    });
-  } catch {
-    verbose(`Unable to parse ${motion.action}`);
-    return undefined;
+  for (const client of clients) {
```
Comment (willm30) [inline src/handlers/motions/motionCreated/helpers.ts:22]: We don't know which client is going to be needed to parse the transaction just from the event data... right?

On this assumption, I'm looping through the set of possible clients and returning the first time we don't get an error.

Comment (rdig) [reply src/handlers/motions/motionCreated/helpers.ts:22]: Correct, and this is the right approach.

We have done this in the past, both for motions and both for normal events

**Review by rdig — APPROVED** [review]
All good

**Review by ArmandoGraterol — APPROVED** [review]
Good one!

File: src/handlers/motions/motionCreated/helpers.ts [hunk 2]
```diff
@@ -14,22 +13,25 @@ import {
 
 export const getParsedActionFromMotion = async (
   motionId: string,
-  colonyClient: AnyColonyClient,
+  colonyAddress: string,
+  clients: NetworkClients[],
```
Comment (jakubcolony) [inline src/handlers/motions/motionCreated/helpers.ts:17]: Would it make sense to restrict this type to only the clients that can parse actions? I think this currently also includes network and token clients

**Review by jakubcolony — APPROVED** [review]
Looking good 👍

**Pull request merged**